wiki/Client/Quick_play/en.md

```diff
@@ -0,0 +1,66 @@
+
+*Note: osu!mania 4K and 7K are split into separate queues and are thus referred to as separate modes.*
+
+## Interface
+
+Users may access quick play by navigating to `play > multi > quick play` via the main menu. The traditional multiplayer lobby interface is accessible via the `lounge` option.
+
+![](img/quick-play-main-menu.png "Main menu interface")
+
+To queue for a match, first select a [mode](/wiki/Game_mode), then click the button to queue for a match.
+
+![](img/quick-play-queue-panel.png "Mode selection")
+
+After users enter the queue, they are brought to the waiting area. This screen shows users who else is queueing simultaneously. User avatars appear as bubbles which orbit and combine as lobbies are formed.
+
+Users may choose to exit this screen and use the client while queueing in the background by clicking the "Queue in background" button.
+
+![](img/quick-play-queue-screen.png "Waiting area")
+
+## Gameplay
+
+The server attempts to form a lobby with 8 players. Smaller lobbies may be encountered should there not be enough players in the queue. There are 5 rounds per match.
+
+### Beatmap selection
+
+Once the lobby is formed, users are presented a small, random set of curated beatmaps to choose from. Beatmaps are voted on by individuals and then picked at random by the server. The more votes a map receives, the more likely it is to be picked. Beatmaps are presented to users based on an estimation of their skill level.
+
+### Matchmaking rating
+
+A hidden skill estimation value called "matchmaking rating" ("MMR", "rating") is assigned to all players and beatmaps. MMR is used to group players with others of a similar skill level. It is also used to influence which beatmaps are chosen by the server for player selection.
+
+A beatmap's MMR is directly derived from it's [star rating](/wiki/Beatmap/Star_rating) (SR). Beatmap curators have the ability to override a beatmap's SR to inform the server of a more accurate difficulty estimation. For example, due to the reading requirement, [this beatmap](https://osu.ppy.sh/beatmapsets/1799413#osu/3688768)'s difficulty (as judged by the curators) is approximately 7.7 stars despite only being ~5.5 stars.
+
+### Scoring
+
+A user's MMR is influenced based on their standing at the end of the lobby. Throughout the match, users are awarded points based on their placement in each round following the table below:
+
+| Placement | Points |
+| :-- | --: |
+| 1st | 15 |
+| 2nd | 12 |
+| 3rd | 10 |
+| 4th | 8 |
+| 5th | 6 |
+| 6th | 4 |
+| 7th | 2 |
+| 8th | 1 |
+
+## Beatmap curation
+
+Beatmaps used in quick play are currently curated by a team of community volunteers, organised by [Stage](https://osu.ppy.sh/users/8191845). The full list of available beatmaps for all modes is publicly available via [this spreadsheet](https://docs.google.com/spreadsheets/d/1ZbvLvHlXH3IF1WgN4YkHqOttO7wG-1Duto-535tqKnQ/edit?gid=0#gid=0).
```

Comment:
In my opinion, if we're explicitly adding the volunteers to the article, we may as well add anyone who's helping as well. Would run it by the people helping out, but the extra transparency will go a long way.

Reply:
I did attempt to add all of the names but quickly realized its too early for that. I'm still not sure who is/isn't sticking around long-term, whether there are leads of each mode like BSC, etc. I mainly included myself so people know who to send beatmap questions to. There are people who have already 'come and gone', not sure what to do about them. I absolutely want to credit the people who help out but I think we need to stabilize first. Also increases PR scope considerably.